javasphinx/apidoc.py

```diff
@@ -192,8 +198,16 @@ def generate_from_source_file(doc_compiler, source_file, cache_dir):
     except Exception:
         util.unexpected('Unexpected exception while parsing %s', source_file)
 
+    documents = {}
     try:
-        documents = doc_compiler.compile(ast)
+        if source_file.endswith("package-info.java"):
```

Comment:
Does something like the following work?

```python
if source_file.endswith("package-info.java"):
    if ast.package is not None:
        documentation = doc_compiler.compile_docblock(ast.package)
        documents[ast.package.name] = (ast.package.name, 'package-info', documentation)
else:
    ...
```

Apologies if my previous comment about the filter was unclear or I'm misunderstanding something.